src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -343,7 +343,7 @@ void func_80853148(PlayState* play, Actor* actor);
 // .bss part 1
 static s32 D_80858AA0;
 static s32 D_80858AA4;
-static Vec3f D_80858AA8;
+static Vec3f sInteractWallCheckResult;
```

Comment:
Same comment here, `sTouchedWallCheckResult`?

Reply:
This is the result of the check it does to see if the touched wall *is* the interact wall. so its the interact-wall-check result